### PR TITLE
Edit Pdf files

### PR DESCRIPTION
**Pull request by NahuelValencia-eb**
Users (ap and supplier) are able to upload new pdfs, when edit information

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -114,3 +114,6 @@ setup_env.sh
 .DS_Store
 fixture.json
 htmlcov
+
+#pdf
```
Comment (leandro-eb) [inline .gitignore]: erase it please

File: supplier_app/tests/test_view.py [hunk 2]
```diff
@@ -1114,6 +1214,55 @@ def test_post_edit_bank_account_info_as_supplier_redirect_to_taxpayer_details(se
...
+        self.assertEqual(
+            self.bank_account.bank_cbu_file,
+            response.context_data['form'].fields['bank_cbu_file'].initial
+        )
+
+    @parameterized.expand([
+        ('supplier', 'supplier@gmail.com'),
+        ('ap_admin', 'fake_ap@eventbrite.com'),
+    ])
+    def test_post_edit_bank_account_view_should_update_bank_account_info(self, group, user):
```
Comment (leandro-eb) [inline supplier_app/tests/test_view.py]: Please be consistent with naming

File: .gitignore (hunk 1 above)
Comment (NahuelValencia-eb) [reply .gitignore]: done

File: supplier_app/tests/test_view.py (hunk 2 above)
Comment (NahuelValencia-eb) [reply supplier_app/tests/test_view.py]: done

**Pull request merged**